pypesto/profile/validation_intervals.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import Optional
+
+from ..engine import Engine
+from ..optimize import Optimizer, minimize
+from ..problem import Problem
+from ..result import Result
+from scipy.stats import chi2
+
+
+logger = logging.getLogger(__name__)
+
+
+def validation_profile_significance(
+        problem_full_data: Problem,
+        result_training_data: Result,
+        result_full_data: Optional[Result] = None,
+        n_starts: int = 1,
+        optimizer: Optional[Optimizer] = None,
+        engine: Engine = None,
+        lsq_objective: float = False
+) -> float:
+    """
+    Computes the significance of an validation experiment as described in
+    Kreutz et al. BMC Systems Biology 2012.
+
+
+    Parameters
+    ----------
+    problem_full_data:
+        pypesto.problem, such that the objective is the
+        negative-log-likelihood of the training and validation data set.
+
+    result_training_data:
+        result object from the fitting of the training data set only.
+
+    result_full_data
+        pypesto.result object, that either contains, or (if fitting was not
```

Comment:
```suggestion
        pypesto.result object that either contains, or (if fitting was not
```
using "that" in the sense of "which", unlike for which there is usually no comma.